### PR TITLE
Add OCR functionality

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
+RUN apt update
+RUN apt -y install tesseract-ocr
+RUN pip install pytesseract
```
Comment (codificat) [inline Dockerfile]: Why not add `pytesseract` and the other required packages to `requirements.txt` instead?

File: Dockerfile [hunk 2]
```diff
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
+RUN apt update
+RUN apt -y install tesseract-ocr
```
Comment (codificat) [inline Dockerfile]: For container image layer economy, a suggestion:
```suggestion
RUN apt update && \
    apt -y install tesseract-ocr tesseract-ocr-nld tesseract-ocr-spa
```

File: Dockerfile [hunk 3]
```diff
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
...
+RUN pip install pytesseract
+
+# an empty directory for test output
+RUN mkdir test_txt
+
+# install languages
+RUN apt-get -y install tesseract-ocr-nld tesseract-ocr-spa
+
+RUN pip install langdetect
+RUN pip install pycountry
```
Comment (codificat) [inline Dockerfile]: I believe these can/should be removed and use `requirements.txt` and a single `apt-get install` instead

**Review by codificat — CHANGES_REQUESTED** [review]
Thanks for this!

Some comments/suggestions

File: Dockerfile (hunk 2 above)
Comment (bn222) [reply Dockerfile]: I prefer adding languages to be on a separate layer to avoid having to rebuild from that step onwards.

**Pull request merged**